node-versions.rb

```diff
@@ -6,5 +6,8 @@
 
 url = URI('https://raw.githubusercontent.com/nodejs/Release/master/schedule.json')
 result = Net::HTTP.get(url)
-lts = JSON.parse(result).select {|key, value| value.key?("lts") and Date.parse(value["start"]) < Date.today }
-lts.each {|key, value| puts "node@#{key.gsub(/v/, '')}" }
+lts = JSON
+  .parse(result)
+  .select {|key, value| value.key?("lts") and Date.parse(value["start"]) < Date.today }
+  .select {|key, value| value.key?("lts") and Date.parse(value["end"]) > Date.today }
```

Comment:
So Homebrew must only suport versions with an `end` date later than the current date.